Assets/Scripts/MouseLook.cs

```diff
@@ -1,26 +1,31 @@
-﻿using UnityEngine;
+﻿using Assets.Scripts.Settings;
+using UnityEngine;
 
 namespace Assets.Scripts
 {
   public class MouseLook : MonoBehaviour
   {
-    public float mouseSensitivity = 100f;
+    SettingsManager _settingsManager;
 
     public Transform playerBody;
 
     public float yRotation;
 
     // Start is called before the first frame update
-    public void Start() => Cursor.lockState = CursorLockMode.Locked;
+    public void Start()
+    {
+      _settingsManager = new SettingsManager();
+      Cursor.lockState = CursorLockMode.Locked;
+    }
 
     // Update is called once per frame
     void Update()
     {
       if (GameService.Instance.IsPaused)
         return;
 
-      var mouseX = Input.GetAxis("Mouse X") * mouseSensitivity * Time.deltaTime;
-      var mouseY = Input.GetAxis("Mouse Y") * mouseSensitivity * Time.deltaTime;
+      var mouseX = Input.GetAxis("Mouse X") * _settingsManager.MouseSensitivity * Time.deltaTime;
```

Comment:
Have you looked into the Input.GetAxis differences on different OSes? You mentioned there was a discrepancy